src/region/RegionAnnotation.tsx

```diff
@@ -16,10 +17,32 @@ const RegionAnnotation = (props: Props, ref: React.Ref<HTMLAnchorElement>): JSX.
     const { isActive, shape, ...rest } = props;
     const { height, width, x, y } = shape;
     const className = classNames('ba-RegionAnnotation', { 'is-active': isActive });
+    const targetRef: React.Ref<SVGRectElement> = React.useRef(null);
+
+    React.useEffect(() => {
+        const scrollOptions: Options = {
```

Comment:
should these default scroll options be defined once above instead of each invocation of this effect?